Cargo.toml

```diff
@@ -93,7 +93,7 @@ rustix = { version = "0.38.44", features = ["fs"] }
 same-file = "1.0.6"
 sapling-renderdag = "0.1.0"
 sapling-streampager = "0.11.0"
-scm-record = { path = "../../arxanas/scm-record/scm-record" }
+scm-record = { git = "https://github.com/jgilchrist/scm-record", branch = "jgilchrist/change/wrvozvmtmssk" }
```

Comment:
We should wait for @arxanas to cut a new release so we can point this to that

Reply:
Yeah, agreed - this was just so that anybody wanting to test this out could just check this out directly